lbrynet/tests/unit/test_conf.py

```diff
@@ -77,3 +79,19 @@ def test_data_dir(self):
         self.assertEqual(str, type(conf.default_download_dir))
         self.assertEqual(str, type(conf.default_data_dir))
         self.assertEqual(str, type(conf.default_lbryum_dir))
+
+    def test_load_save_load_config_file(self):
+        #settings = self.get_mock_config_instance()
+        conf_entry = 'lbryum_servers: ["localhost:50001", "localhost:50002"]\n'
+        conf_temp = create_conf_file(conf_entry)
+        conf.conf_file = conf_temp
+        adjustable_settings={'data_dir': (str, conf.default_data_dir),
+                'lbryum_servers': (list, [('localhost', 5001)],
+                    conf.server_list, conf.server_list_reverse)}
+        env = conf.Env(**adjustable_settings)
+        settings = conf.Config({}, adjustable_settings, environment=env)
+        conf.settings = settings
+        settings.load_conf_file_settings()
+        settings.save_conf_file_settings()
+        settings.load_conf_file_settings()
```

Comment:
The test needs to assert something. This just proves that no exceptions are thrown, but doesn't guarantee that the desired behavior is occurring.